### PR TITLE
Angular template for config app.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Ready for first review.

My Iast TODOs:
* updated config (requires #1529)
* reset all configs to default values

@normanjaeckel I have some suggestions for you to update the rest-api:
* use new prefix "general_" for all general config keys
* introduce new "type" value for each config field to show the correct html input type in list.html (e.g. radio, checkbox, text, number etc.; see http://www.w3schools.com/html/html_form_input_types.asp)
* rename config names: "motion" -> "motions"; "assignment" -> "assignments"

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@normanjaeckel A "label" and a optional "help" field would be nice for each config field to show config forms like in OpenSlides 1.x

**Comment by ostcar** [comment]
@emanuelschuetze to fix travis, you only have to wait for #1539 and rebase on top of it (which should be easy, because you did not touch so much python code)

**Comment by normanjaeckel** [comment]
See #1545 for new config var names. New type is still missing.

**Comment by emanuelschuetze** [comment]
Rebased. I'll now add the update function.
@normanjaeckel Please add a issue/pull request with your last TODO (add new type).

File: openslides/urls.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
     url(r'^', include('openslides.core.urls')),
     url(r'^agenda/', include('openslides.agenda.urls')),
     url(r'^assignments/', include('openslides.assignments.urls')),
+    url(r'^config.*', IndexView.as_view()),
```
Comment (ostcar) [inline openslides/urls.py]: this line can be removed, because we have a fallback to to IndexView now.

**Comment by normanjaeckel** [comment]
Ok.

**Comment by emanuelschuetze** [comment]
Ready for review.

**Comment by ostcar** [comment]
@emanuelschuetze one note, then it can be merged

**Comment by ostcar** [comment]
This pull request should be rebased ontop of #1549 because it will break after this pull request is merged.

**Comment by emanuelschuetze** [comment]
Fix one remark. Rebased. I'll wait until #1549 is merged to update and rebase.

**Comment by emanuelschuetze** [comment]
@ostcar I'll wait for your pullrequest for #1558.

**Comment by emanuelschuetze** [comment]
Rebased and added code to reset to default_value.
Still open: We have to check if the input is in right input type _before_ sending PUT request.. Otherwise the PUT-Request failed and shows some more errors.

**Comment by normanjaeckel** [comment]
@emanuelschuetze: You may want this to be merged and move the last hint to an extra issue?

**Comment by emanuelschuetze** [comment]
If @ostcar can do the last pull request (or give me some hints to do that) in the next days I will wait for it and merge it together... Currently my solution does not work for e.g. config choices.

**Comment by ostcar** [comment]
@emanuelschuetze it would be easier for me if you rebased and merged this (unfinished) pull request. Then I can fix the config field on top of it.

**Comment by ostcar** [comment]
@emanuelschuetze Please move the code in config/static to core/static

**Comment by emanuelschuetze** [comment]
Moved code from config/static to core/static. Rebased.
Ready for review and merge.

**Pull request merged**